### ISSUE TITLE
[Bug]: [a11y] Inconsistency between how the wallpaper snackbars are announced with talkback

### ISSUE THREAD

**Issue by delia-pop** [mozilla-mobile/fenix#27132]
### Steps to reproduce

1. Turn TalkBack (or the available screen reader) on.
2. From the Wallpapers Setting screen double tap on a thumbnail.
3. Tap on the "Wallpaper changed! VIEW" snackbar. Observe.
4. Disconnect from network. 
5. Tap on different thumbnail to trigger the error snackbar.
6. Tap on the "Couldn't download wallpaper. Try Again" snackbar. Observe.

### Expected behaviour

Both snackbars are announced in the same way.

### Actual behaviour

After Step 3, the Talkback announces "Wallpaper updated", without announcing the button.
After Step 6, the Talkback announces "Couldn't download wallpaper. Try Again **button**".

Note that on both snackbars the button is announced separately when explicitly tapped.

### Device name

Google Pixel 6 

### Android version

Android 13

### Firefox release type

Firefox Nightly

### Firefox version

Nightly 107.0a1 from 09/23

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-21650)

**Comment by cpeterso** [comment]
Not a blocker for MR v106.

**Comment by mcarare** [comment]
To me, this seems as expected, because in the first use case the focus is taken by the new wallpaper selection, information that takes precedence over the content of the ephemeral message (the snackbar) because is more important to the user. 

I would even say that in some cases (depending on OS/ Talkback version) the confirmation snack bar would not be read at all. For a more detailed answer, the fact that the snackbar text might not be read is due to the fact that the snackbar view is set as an **accessibility live region** with **polite** priority. That means it is important to the user, but not that important to determine the interruption of other announcements. 

Note that when selecting the snackbar, the text is read the same in both cases.

**Comment by cpeterso** [comment]
> To me, this seems as expected, because in the first use case the focus is taken by the new wallpaper selection, information that takes precedence over the content of the ephemeral message (the snackbar) because is more important to the user.

@mcarare In that case, do you think we should close this bug as WONTFIX or not a bug?

**Comment by mcarare** [comment]
@cpeterso Yes, this can be closed as WONTFIX.

**Comment by cpeterso** [comment]
Thanks. In that case, I'll close this issue.
